pdg/src/snapshots/c2rust_pdg__tests__analysis_test_pdg_snapshot.snap

```diff
@@ -3,999 +3,999 @@ source: pdg/src/main.rs
+	n[20]: copy       n[19] => _22 @ bb7[7]:  fn lighttpd_test; _22 = move _23 as *mut libc::c_void (Misc)                                    
+	n[21]: free       n[20] => _21 @ bb7[8]:  fn lighttpd_test; _21 = pointers::free(move _22)                                                
 }
 nodes_that_need_write = [3, 2, 1, 0]
 
 g {
-	n[0]: &_11 _ => _10 @ bb4[14]: fn lighttpd_test;
+	n[0]: &_11 _ => _10 @ bb4[14]: fn lighttpd_test; _10 = &mut _11
 }
 nodes_that_need_write = []
 
 g {
-	n[0]: &_10      _    => _14 @ bb4[18]: fn lighttpd_test;      
-	n[1]: copy      n[0] => _1  @ bb0[0]:  fn connection_accepted;
-	n[2]: field.0   n[1] => _10 @ bb2[10]: fn connection_accepted;
-	n[3]: addr.load n[2] => _   @ bb2[10]: fn connection_accepted;
+	n[0]: &_10      _    => _14 @ bb4[18]: fn lighttpd_test; _14 = &raw mut (*_10)                                           
+	n[1]: copy      n[0] => _1  @ bb0[0]:  fn connection_accepted; _13 = pointers::connection_accepted(move _14, const 0_i32)
+	n[2]: field.0   n[1] => _10 @ bb2[10]: fn connection_accepted; _10 = ((*_1).0: *mut pointers::fdevents)                  
+	n[3]: addr.load n[2] => _   @ bb2[10]: fn connection_accepted; _10 = ((*_1).0: *mut pointers::fdevents)                  
 }
 nodes_that_need_write = []
 
 g {
-	n[0]:  alloc       _     => _5      @ bb1[2]:  fn connection_accepted;
-	n[1]:  copy        n[0]  => _4      @ bb2[2]:  fn connection_accepted;
-	n[2]:  field.0     n[1]  => _       @ bb2[6]:  fn connection_accepted;
-	n[3]:  addr.store  n[2]  => _       @ bb2[6]:  fn connection_accepted;
-	n[4]:  field.0     n[1]  => _11     @ bb2[12]: fn connection_accepted;
-	n[5]:  addr.load   n[4]  => _       @ bb2[12]: fn connection_accepted;
-	n[6]:  copy        n[1]  => _15     @ bb2[21]: fn connection_accepted;
-	n[7]:  copy        n[6]  => _14     @ bb2[22]: fn connection_accepted;
-	n[8]:  copy        n[7]  => _4      @ bb0[0]:  fn fdevent_register;   
-	n[9]:  copy        n[8]  => _15     @ bb2[16]: fn fdevent_register;   
-	n[10]: value.store n[9]  => _12.*.1 @ bb2[17]: fn fdevent_register;   
-	n[11]: field.1     n[1]  => _       @ bb3[4]:  fn connection_accepted;
-	n[12]: addr.store  n[11] => _       @ bb3[4]:  fn connection_accepted;
-	n[13]: copy        n[8]  => _0      @ bb3[7]:  fn connection_accepted;
-	n[14]: copy        n[13] => _13     @ bb5[0]:  fn lighttpd_test;      
-	n[15]: copy        n[14] => _17     @ bb5[7]:  fn lighttpd_test;      
-	n[16]: copy        n[15] => _2      @ bb0[0]:  fn connection_close;   
-	n[17]: field.1     n[16] => _5      @ bb0[4]:  fn connection_close;   
-	n[18]: addr.load   n[17] => _       @ bb0[4]:  fn connection_close;   
-	n[19]: field.0     n[16] => _8      @ bb1[7]:  fn connection_close;   
-	n[20]: addr.load   n[19] => _       @ bb1[7]:  fn connection_close;   
+	n[0]:  alloc       _     => _5      @ bb1[2]:  fn connection_accepted; _5 = pointers::malloc(move _6)                                      
```

Comment:
The output seems to produce a lot of unnecessary whitespace at end of line in some cases

Reply:
thanks, this was a bug